bp_me/test/tb/bp_cce/Makefile.params

```diff
@@ -47,8 +44,7 @@ export TB_PARAMS  = \
 
 export DUT_DEFINES =
 export TB_DEFINES  = +define+BP_SIM_CLK_PERIOD=$(BP_SIM_CLK_PERIOD) \
-                     +define+dram_pkg=$(DRAM_PKG) \
-                     +define+$(DRAM_MODEL)
+					 +define+den2048Mb+sg5+x16+FULL_MEM
```

Comment:
do we not want to be able to change the DRAM_MODEL ever? (question applies to all testbenches)

Reply:
I think we only have the one micron model